CRM/Event/Form/Participant.php

```diff
@@ -1740,8 +1740,6 @@ public function buildEventFeeForm($form) {
     }
     $form->assign('onlinePendingContributionId', $form->get('onlinePendingContributionId'));
 
-    $form->assign('paid', $form->_isPaidEvent);
```

Comment:
This removal causes a problem since now it never shows the fee selections!

Reply:
We seem to be back to the fees not appearing again. I see this is moved up to the top, but `$form->_isPaidEvent` isn't set until after that.